ledger/acctonline_test.go

```diff
@@ -329,9 +329,9 @@ func TestAcctOnlineCache(t *testing.T) {
 
 	const seedLookback = 2
 	const seedInteval = 3
-	const maxBalLookback = 2 * seedLookback * seedInteval
-
 	const numAccts = 5
+	const maxBalLookback = 3 * numAccts
```

Comment:
please keep the old formula for maxBalLookback

Reply:
why?

Reply:
This does not matter much in ledger unit tests but it matter in e2e and node tests. So keeping the formula the same in all places plays on consistency side. If you want to disconnect (and this is fine in unit tests), get rid of seedLookback and seedInteval as well.